src/utils/api.js

```diff
@@ -0,0 +1,84 @@
+class Api {
+  constructor({baseUrl, headers}) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    this.getUserInfo();
```

Comment:
` this.getUserInfo();` is not needed here. You have it in line 9

Reply:
This has been deleted